src/styles/utility.css

```
@@ -1,5 +1,5 @@
 .flow > * + * {
-  margin-top: 2rem;
+  margin-top: var(--spacing-l, var(--flow-spacing));
```

Did you mean to do this ?
```suggestion
  margin-top: var(--flow-spacing, var(--spacing-l));
```

---

Yes!